src/test/java/com/wikia/webdriver/testcases/mercurytests/old/PortableInfoboxTests.java

```diff
@@ -96,18 +94,16 @@ public void mercury_infobox_externalLinkRedirectsToCorrespondingUrl() {
     infobox.isExternalLinkLabelInURL(externalLinkName, externalURL);
   }
 
-  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox", enabled = false)
-  @RelatedIssue(issueID = "XW-1200", comment = "functionality is broken")
+  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox")
   public void mercury_infobox_imagesAndVideosOpenInLightbox() {
     init();
     navigate.toPage(MercurySubpages.INFOBOX_1);
 
     // Check image
     infobox
-        .clickHeroImage()
+        .clickOnImageInInfobox()
         .isLightboxOpened()
-        .closeLightbox()
-        .isTitleOverImageVisible();
```

Comment:
that is true, but we don't have to do it here, we have different test for checking lightbox functionality, now we want to focus on infobox only :)